consoles/vnc_base.pm

```diff
@@ -192,31 +169,17 @@ sub mouse_hide {
     return {absolute => $self->{vnc}->absolute};
 }
 
-sub mouse_set {
-    my ($self, $args) = @_;
+sub mouse_set ($self, $args) {
     die "Need x/y arguments" unless (defined $args->{x} && defined $args->{y});
-
-    # TODO: for framebuffers larger than 1024x768, we need to upscale
     $self->_mouse_move(int($args->{x}), int($args->{y}));
     return {};
 }
 
-sub mouse_button {
-    my ($self, $args) = @_;
-
+sub mouse_button ($self, $args) {
     my $button = $args->{button};
     my $bstate = $args->{bstate};
-
-    my $mask = 0;
-    if ($button eq 'left') {
-        $mask = $bstate;
-    }
-    elsif ($button eq 'right') {
-        $mask = $bstate << 2;
-    }
-    elsif ($button eq 'middle') {
-        $mask = $bstate << 1;
-    }
+    my %mask = (left => $bstate, right => $bstate << 2, middle => $bstate << 1);
+    my $mask = $mask{button} // 0;
```

Comment:
As pointed out by @perlpunk in the other PR, this misses a `$`.

Reply:
@okurz  it's really confusing to have the same commits in several open pull requests.
Good that I didn't start to review this one yet.

Reply:
That's way I cross-reference PRs. The number of commits can also be an indication. Start with the ones with least commits

Reply:
> > Start with the ones with least commits
> 
> No, I refuse to. I requested changes on #1914 and I will not go over the commits in several other PRs to request the same changes.

Sure, fine. I am happy about review feedback either way.

> Please mark your PRs in a way that people don't have to figure that out themselves.

Sorry, I try to. The description in https://github.com/os-autoinst/os-autoinst/pull/1914 says "Based on #1913 and #1789". I thought that makes it clear. Maybe together we can find a better general approach.

Anyway, PR updated to include your proposals in this PR as well as the other.

Reply:
It is still missing the `$` sign.

Reply:
oh, geez. That's embarrassing. Sorry about that. Fixed